.github/blunderbuss.yml

```diff
@@ -0,0 +1,2 @@
+assign_prs:
+  - datacommonsorg/core
```

Comment:
Thank you for adding this!  Would it be worth having a different team, with more focussed reviewers? (e.g., Core has Rafik, Guha)